web/src/engine/predictive-text/worker-thread/src/main/correction/context-tokenization.ts

```diff
@@ -735,4 +750,177 @@ export function traceInsertEdits(tokens: string[], transform: Transform): {
     stackedInserts,
     firstInsertPostIndex
   };
+}
+
+/**
+ * Given context tokenizations before and after applying an incoming transform,
+ * this method analyzes the tokenizations in order to make adjustments for any
+ * 'merge' or 'split' edits needed for the transition.
+ * @param priorTokenization
+ * @param resultTokenization
+ * @returns
+ */
+export function analyzePathMergesAndSplits(priorTokenization: string[], resultTokenization: string[]): {
+  /**
+   * Notes the shift in pre-edit token index needed to map the tail token index
+   * to its post-edit index due to 'merge' edits.
+   */
+  mergeOffset: number,
+  /**
+   * Notes the shift in post-edit token index needed to map the tail token index
+   * to its pre-edit index due to 'split' edits.
+   */
+  splitOffset: number,
+  /**
+   * The edit operations needed to transition from the first tokenization to the second.
```

Comment:
Are the `priorTokenization` and `resultTokenization` parameters the first and second tokenization, or is that something else?